libs/sdk-ui-ext/src/internal/dashboardEmbedding/ScheduledMail/utils/test/scheduledMailSummary_test.ts

```diff
@@ -0,0 +1,197 @@
+// (C) 2019-2020 GoodData Corporation
```

Comment:
Please use the naming convention of `scheduledMailSummary.test.ts` used in the rest of the repo (notice the dot). This is present in all the newly added tests in this PR.

Reply:
Oh, I missed utils folder :) thanks